engines/groovie/script.cpp

```diff
@@ -959,8 +959,8 @@ bool Script::playvideofromref(uint32 fileref, bool loopUntilAudioDone) {
 			Common::String subtitleName = _vm->_resMan->getGjdName(info);
 			subtitleName = subtitleName.substr(0, subtitleName.size() - 4);
 			subtitleName.toUppercase();
-			// add the filename without the extension, then add the .txt extension
-			subtitleName += "-" + info.filename.substr(0, info.filename.size() - 3) + "txt";
+			// add the filename without the extension, then add the srt extension
+			subtitleName += "-" + info.filename.substr(0, info.filename.size() - 3) + "srt";
```

Comment:
The actual subtitles pack contained .txt as the subtitle extension.

Reply:
Is it something already published? I thought it was still in testing phase.
If it's still testing, I think it would be better to stick to srt extension.

Reply:
You are right, now I remember that it was internal release